sources/config_reader.c

```diff
@@ -1729,10 +1730,28 @@ static int od_config_reader_route(od_config_reader_t *reader, char *db_name,
 
 static inline int od_config_reader_watchdog(od_config_reader_t *reader,
 					    od_storage_watchdog_t *watchdog,
+					    char *storage_name,
 					    od_extention_t *extentions)
 {
-	watchdog->route_usr = "watchdog_int";
-	watchdog->route_db = "watchdog_int";
+	char *watchdog_prefix = "watchdog_";
+	int watchdog_prefix_len = strlen(watchdog_prefix);
+	int storage_name_len = strlen(storage_name);
+
+	char route_usr[watchdog_prefix_len + storage_name_len + 1];
+	char route_db[watchdog_prefix_len + storage_name_len + 1];
+	snprintf(route_usr, sizeof route_usr, "%s%s", watchdog_prefix,
+		 storage_name);
+	snprintf(route_db, sizeof route_db, "%s%s", watchdog_prefix,
+		 storage_name);
+
+	watchdog->route_usr = strdup(route_usr);
```

Comment:
I hope we do free to this allocations...

Reply:
fixed